docs/guides.md

```diff
@@ -523,3 +523,307 @@ Now we enable autostart at the virtual machine level.
+
+#### Different Sized Drives
+
+We might need to create a RAID array where our drives are not identical and each drive has a different number of available blocks.  This might come up if we need to create a RAID array but have two of one type of drive and one of another such as two WD drives and one Seagate or two 1TB drives and one that is 1.5TB or 2TB.
+
+The easiest solution to creating a working RAID array in this situation is to partition the drives and create a RAID array using the partitions instead of using the entire drive.
+
+To do this, we get the sizes of the disks in the system by examining the /proc/partitions file.  Starting with the smallest of the disks to be used in the array, use gdisk or sgdisk to create a single partition of type fd00 (Linux RAID) using the maximum space available.  Examine and record the size of the partition created and save the changes.  Repeat the process with the remaining drives to be used except use the size of the partition created on the first drive instead of the maximum space available.
+
+This should leave you with drives that each have a single partition and all of the partitions are the same size even though the drives are not.
+
+When creating the RAID array and the mdadm.conf file, use the name of the disk partition instead of the name of the disk.  In our example system, we would create the array using /dev/sdc1, /dev/sdd1, and /dev/sde1 instead of /dev/sdc, /dev/sdd, and /dev/sde and also make the same substitutions on DEVICE lines in the mdadm.conf file.
+
+It should also be possible to create the partitions on the drives outside of the XCP-ng system using a bootable utility disk that contains partitioning utilities such as gparted.
+
+#### More Than One Additional Array
+
+We might want to create more than one extra RAID array and storage repository.  This is also easy to accomodate in a similar way to using a different number of drives in the array.  We can easily create another RAID array and another storage repository onto a different set of drives by changing the parameters of the mdadm --create command line and xe sr-create command line.
+
+As an example assume that we have 3 more drives /dev/sdf, /dev/sdg, and /dev/sdh and want to create a second RAID 5 array and another storage repository.  We create another RAID 5 array, this time md1 like this:
+
+```
+[16:45 XCP-ng ~]# mdadm --create /dev/md1 --run --level=5 --bitmap=internal --assume-clean --raid-devices=3 --metadata=1.2 /dev/sdf /dev/sdg /dev/sdh
+mdadm: array /dev/md1 started.
+```
+
+We then create another storage repository as we did previously making sure to give it a different name and use /dev/md1 instead of /dev/md0 in the command line.
```

Comment:
```suggestion
We then create another storage repository as we did previously making sure to give it a different name and use `/dev/md1` instead of `/dev/md0` in the command line.
```